src/components/pages/quickstart/Prerequisites/python.mdx

```diff
@@ -54,14 +54,13 @@ client = Arcade(
     ARCADE_API_KEY=your_api_key_here
     ```
 
-    3. Load the API key in your code:
+    3. Load the `.env` file in your code:
 
     ```python
     from dotenv import load_dotenv
-    import os
 
     load_dotenv()
-    api_key = os.getenv("ARCADE_API_KEY")
+    client = Arcade() # the env var ARCADE_API_KEY is loaded automatically
```

Comment:
Makes it clear that once the env var is set (either with a `.env` file, or not), the client will find it automatically.